splunk/src/main/java/com/splunk/HttpService.java

```diff
@@ -73,7 +80,7 @@ public boolean verify(String s, SSLSession sslSession) {
     private String prefix = null;
 
     static Map<String, String> defaultHeader = new HashMap<String, String>() {{
-        put("User-Agent", "splunk-sdk-java/1.6.4");
+        put("User-Agent", "splunk-sdk-java/1.7.0");

```

Comment:
Nice, yes this is the header that's required for telemetry to track usage/adoption of new SDK versions.

Reply:
Okay.